--- ./Documents/Arduino_OLD/libraries/Etherkit_JTEncode/src/JTEncode.cpp	2021-07-19 22:47:32.000000000 +0200
+++ ./Arduino/libraries/Etherkit_JTEncode/src/JTEncode.cpp	2023-04-27 23:23:37.123054287 +0200
@@ -997,6 +997,24 @@
 			strncpy(prefix, callsign, slash_pos);
 			strncpy(base_call, callsign + slash_pos + 1, 7);
 
+#if 1   //PE0FKO Update one char prefix
+			if(prefix[1] == ' ' || prefix[1] == 0)
+			{
+				// Right align prefix
+				prefix[3] = 0;
+				prefix[2] = prefix[0];
+				prefix[1] = ' ';
+				prefix[0] = ' ';
+			}
+			else if(prefix[2] == ' ' || prefix[2] == 0)
+			{
+				// Right align prefix
+				prefix[3] = 0;
+				prefix[2] = prefix[1];
+				prefix[1] = prefix[0];
+				prefix[0] = ' ';
+			}
+#else
 			if(prefix[2] == ' ' || prefix[2] == 0)
 			{
 				// Right align prefix
@@ -1005,6 +1023,7 @@
 				prefix[1] = prefix[0];
 				prefix[0] = ' ';
 			}
+#endif
 
 			for(uint8_t i = 0; i < 6; i++)
 			{
